pkg/mx/handler.go

```diff
@@ -440,6 +553,14 @@ func podList(pods v1.PodList) (ret []*v1.Pod) {
 	return
 }
 
+func serviceList(services v1.ServiceList) (ret []*v1.Service) {
+	for _, s := range services.Items {
```

Comment:
`ret` could be prior allocated with `len(services.Items)`.

Reply:
pointer type, ignore overhead